src/main/java/com/deviantart/kafka_connect_s3/S3SinkTask.java

```diff
@@ -55,29 +54,32 @@ public void start(Map<String, String> props) throws ConnectException {
         // keep default
       }
     }
-    String bucket = config.get("s3.bucket");
-    String prefix = config.get("s3.prefix");
+    String bucket = config.get(S3SinkConnectorConstants.S3_BUCKET_CONFIG);
+    String prefix = config.get(S3SinkConnectorConstants.S3_PREFIX_CONFIG);
     if (bucket == null || bucket == "") {
       throw new ConnectException("S3 bucket must be configured");
     }
     if (prefix == null) {
       prefix = "";
     }
 
+    String s3Region = config.get(S3SinkConnectorConstants.S3_REGION_CONFIG);
```

Comment:
Whats the default here? We probably want this to be optional, so dont set the region if there is not a config key.